### PR TITLE
[FLINK-20274][docs-zh] Translate page 'testing' into Chinese

### PR DESCRIPTION
**Pull request by metaotao**
## What is the purpose of the change

Translate page "testing" into Chinese.

the page url is https://ci.apache.org/projects/flink/flink-docs-master/zh/dev/stream/testing.html.

The doc located in "flink/docs/dev/stream/testing.zh.md"

## Brief change log

- translate "flink/docs/dev/stream/testing.zh.md"

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

**Comment by metaotao** [comment]
@flinkbot run azure

**Comment by metaotao** [comment]
Hi, @wuchong , Could you help review this PR in your free time?  thanks.

**Comment by metaotao** [comment]
Hi, @fsk119 , Could help review this PR? thanks.

**Comment by fsk119** [comment]
Sure. 
Thanks for your contribution. I will review it in the Monday.

File: docs/dev/stream/testing.zh.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ class PassThroughProcessFunction extends ProcessFunction[Integer, Integer] {
 </div>
 </div>
 
-It is very easy to unit test such a function with `ProcessFunctionTestHarnesses` by passing suitable arguments and verifying the output.
+通过传递合适的参数并验证输出，对使用 `ProcessFunctionTestHarnesses` 这样的函数进行单元测试是非常容易的。
```
Comment (fsk119) [inline docs/dev/stream/testing.zh.md:367]: 通过传递合适的参数，使用`ProcessFunctionTestHarnesses` 是很容易进行单元测试并验证输出。

**Review by fsk119 — COMMENTED** [review]
I left some comments.

**Comment by metaotao** [comment]
Hi, @fsk119 , Thank you for your detailed review, it has been updated~

**Review by fsk119 — APPROVED** [review]
LGTM

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**